sys/include/event/periodic.h

```diff
@@ -91,9 +111,29 @@ static inline void event_periodic_start(event_periodic_t *event_periodic,
  * connected event will be put already into the given event queue and this
  * function does not have any effect.
  *
+ * The count value is reset to EVENT_PERIODIC_FOREVER.
```

Comment:
Why though?

Reply:
So that is `start` is called again it will again start running forever, that is why I added `event_periodic_pause` which does not reset that, I thought having an explicit function made it less ambiguous, while keeping the same behaviour for the current api users.

Reply:
But current API users would always have it at `EVENT_PERIODIC_FOREVER`, so even without a change to `event_periodic_stop()` the behavior would stay the same.

Reply:
> I'm not objecting the new function, I just can't quite think of a use case.

I have one but in a bit of a weird scenario, and I wasn't actually using this function. It was more that after the latest changes I was wondering ok if `set_count` was called and the event then stopped, then what should happen on the next call to start? Allowing for both allowed me to skip answering the question hahaha

Reply:
That sounds like a weak use-case :wink: 

How about `event_periodic_stop()` does not touch the `count` variable (it acts just like `pause`). If the user wants to change the `count` value, they can still call `event_periodic_set_count()`.

Reply:
Ok!